src/jmh/java/column/store/query/span/FilterForSpanId.java

```diff
@@ -0,0 +1,101 @@
+package column.store.query.span;
+
+import column.store.Utils;
+import column.store.api.column.Column;
+import column.store.api.column.IdColumn;
+import column.store.api.query.Query;
+import column.store.api.read.Reader;
+
+import static column.store.api.query.Filter.whereId;
+import static column.store.util.Conditions.checkState;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.infra.Blackhole;
+
+@Measurement(iterations = 3)
+@Warmup(iterations = 2)
+@Fork(1)
+public class FilterForSpanId {
+
+  private static final IdColumn SPAN_ID = Column.forId("span_id-ID_64");
+  private static final byte[] SPAN_ID_PRESENT = new byte[] { -120, -84, -113, 10, -114, -22, 16, 104 };
+  private static final byte[] SPAN_ID_NOT_PRESENT = new byte[] { 0, 0, 0, 0, 0, 0, 0 ,0 };
+
+  @State(Scope.Thread)
+  public static class BenchState {
+
+    @Param({ "parquet", "csv" })
+    public String readerType;
+//    @Param({ "true", "false" })
```

Comment:
Note that stable / unstable has yet to be enabled

Reply:
ok no problem.
out of interest: why ?